config/local-setup/dns-operator/kustomization.yaml

```diff
@@ -8,3 +8,9 @@ patches:
       kind: CustomResourceDefinition
       metadata:
         name: dnsrecords.kuadrant.io
+  - patch: |-
```

Comment:
Not getting why this is needed?

Reply:
The `../../deploy/local` kustomization which this is modifying adds the crds which prevents you from using it multiple times in an overlay. When we install multiple instances of the dns operator we can use this kustomization which removes the CRDs multiple times, and install the crds separately once.